io.elementary.code.json

```diff
@@ -1,12 +1,12 @@
 {
-	"app-id":"io.elementary.code",
-	"runtime":"org.gnome.Platform",
-	"runtime-version":"3.24",
-	"sdk":"org.gnome.Sdk",
-	"command":"scratch-text-editor",
-	"rename-desktop-file":"org.pantheon.scratch.desktop",
-	"rename-appdata-file":"org.pantheon.scratch.appdata.xml",
-	"finish-args":[
+	"app-id": "io.elementary.code",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "scratch-text-editor",
+	"rename-desktop-file": "org.pantheon.scratch.desktop",
```

Comment:
Not important for this PR but do you have any idea if the Elementary folks are interested in renaming their data files upstream? It would be quite nice if they got on board that.